src/test/java/org/wildfly/security/ldap/DirContextFactoryRule.java

```diff
@@ -75,8 +208,15 @@ public void evaluate() throws Throwable {
     public ExceptionSupplier<DirContext, NamingException> create() {
         SocketFactory socketFactory;
         try {
+            try {
+                File workingDirCA = new File(CA_JKS_LOCATION);
+                File trustFile = new File(workingDirCA, "ca.truststore");
+                if (trustFile.exists() == false) {
+                    setUp();
+                }
+            } catch (Exception e) {}
```

Comment:
The same thing would happen with this empty catch block as well. Since the outer try/catch block already catches ```Exception```, this inner try/catch block could just be removed.

Reply:
Fixed